### ISSUE TITLE
how to use it?

### ISSUE THREAD

**Issue by longjianquan** [windelbouwman/move-base-ompl#8]
i have compile this project.but how to us it?roslaunch turtlebot_stage turtlebot_in_stage? i have add <param name="base_global_planner" value="ompl_global_planner/OmplGlobalPlanner"/> in move_base.launch.xml    .
but when i launch roslaunch turtlebot_stage turtlebot_in_stage
 it show 
[FATAL] [1520940319.066757250, 1.800000000]: Failed to create the ompl_global_planner/OmplGlobalPlanner planner, are you sure it is properly registered and that the containing library is built? Exception: Failed to load library /home/ljq/catkin_ws/devel/lib//libompl_global_planner_lib.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library (Poco exception = /home/ljq/catkin_ws/devel/lib//libompl_global_planner_lib.so: undefined symbol: _ZNK4ompl4base21OptimizationObjective11isSatisfiedENS0_4CostE)
[move_base-10] process has died [pid 6257, exit code 1, cmd /opt/ros/indigo/lib/move_base/move_base cmd_vel:=navigation_velocity_smoother/raw_cmd_vel odom:=odom scan:=scan __name:=move_base __log:=/home/ljq/.ros/log/3415c6a6-26b1-11e8-b9e4-1c1b0daf1897/move_base-10.log].
log file: /home/ljq/.ros/log/3415c6a6-26b1-11e8-b9e4-1c1b0daf1897/move_base-10*.log
    

how to solov it? thanks

**Comment by longjianquan** [comment]
i have found the solution.
  the CMakelists.txt     should add a line   (target_link_libraries(ompl_global_planner_lib ${OMPL_LIBRARIES}))

https://answers.ros.org/question/224971/cannot-load-ompl-global-planner-plugin/

**Comment by windelbouwman** [comment]
I update the code. Does this solve the issue? Is this issue duplicate of #5 ?

**Comment by longjianquan** [comment]
yes  i have solve it. Thank you

**Comment by longjianquan** [comment]
how to get ROS grid map cell information?how to get ROS map obstacle?

**Comment by windelbouwman** [comment]
What do you mean? Can you describe in more words what you want to do?

**Comment by windelbouwman** [comment]
Btw, was this issue a duplicate of #5?

**Comment by longjianquan** [comment]
i have send the error in your email. notifications@github.com is your address?

**Comment by windelbouwman** [comment]
No, that is not my mail address. Please use this issue tracker to put your question in.

**Comment by longjianquan** [comment]
but i want to show the pictures which run in ROS and found questions

**Comment by xujiajie1997** [comment]
I followed this and still the same error why？
